packages/component-base/src/focus-trap-controller.js

```diff
@@ -56,6 +58,8 @@ export class FocusTrapController {
   trapFocus(trapNode) {
     this.__trapNode = trapNode;
 
+    instances.push(this);
```

Comment:
I guess it would make sense to place this after the `this.__focusableElements.length === 0` check.

Reply:
Thanks, done.